README.md

```diff
@@ -1,220 +1,172 @@
+MCLC by default only has Vanilla. In order to automate the process of installing [Forge](https://minecraftforge.net), [Fabric](https://fabricmc.net) and [Quilt](https://quiltmc.org) you would need to use lorem ipsum
 
-##### refreshAuth 
+### Documentation
 
-| Parameter          | Type   | Description                                                                         | Required |
-|--------------------|--------|-------------------------------------------------------------------------------------|----------|
-| `access_token`     | String | Token being checked if it can be used to login with (online mode).                  | True     |
-| `client_token`     | String | Token being checked if it's the same client that the access_token was created from. | True     |
+#### Client Functions
 
-##### invalidate
+<!-- will fill in later-->
 
-| Parameter    | Type   | Description                                                       | Required |
-|--------------|--------|-------------------------------------------------------------------|----------|
-| `access_token` | String | Token being checked if it can be used to login with (online mode). | True     |
-| `client_token` | String | Token being checked if it's the same client that the access_token was created from. | True     |
+| Function | Type    | Description                                                                                |
+| -------- | ------- | ------------------------------------------------------------------------------------------ |
+| `launch` | Promise | Launches the client with the specified `options` as a parameter. Returns child the process |
 
-##### signOut
+##### launch
 
-| Parameter    | Type   | Description                          | Required |
-|--------------|--------|--------------------------------------|----------|
-| `username` | String | Username used to login with | True     |
-| `password` | String | Password used to login with | True     |
+<!-- will fill in later-->
+
+| Parameter                      | Type    | Description                                                                                                                      | Required |
+| ------------------------------ | ------- | -------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `options.removePackage`        | Boolean | Option to remove the client package zip file after its finished extracting.                                                      | False    |
+| `options.root`                 | String  | Path where you want the launcher to work in. `C:/Users/user/AppData/Roaming/.mc`                                                 | True     |
+| `options.cache`                | String  | Path where launcher files will be cached in. `C:/Users/user/AppData/Roaming/.mc/cache`                                           | False    |
```

Comment:
Why don't you use the official Minecraft resources (if they are present) to cache resources?